### PR TITLE
Arch doc

### PR DESCRIPTION
**Pull request by kasimrehman**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -22,16 +27,25 @@ Each service on this full list of services included in the repository has a desi
 - [onboarding-skill](markdown/onboarding-skill.md)
 - [storage-adapter-mongodb](markdown/storage-adapter-mongodb.md)
 
+### Architecture
+
+The following shows the services in a [TAM](http://www.fmc-modeling.org/fmc-and-tam) block diagram.
+
+![The big picture](images/AAS_SERVICE_REVISED.png).
```
Comment (Manu3756) [inline docs/README.md]: Image is not displayed

Comment (kasimrehman) [reply docs/README.md]: It is not displayed in the source code but in the actual link it shows for me: https://github.com/SAP/i40-aas/tree/arch-doc/docs. Is this what you meant?

Comment (kasimrehman) [reply docs/README.md]: It is not displayed in the source code but in the actual link it shows for me: https://github.com/SAP/i40-aas/tree/arch-doc/docs. Is this what you meant?

Comment (Manu3756) [reply docs/README.md]: ok when I click on the link I get a 404 from Github. Maybe a bug in the UI of Github itself.

Comment (kasimrehman) [reply docs/README.md:34]: I am not sure what changes to perform here.

[the remark this answers is not in the document]
Comment (kasimrehman) [reply docs/README.md:34]: The storage components are part of TAM. They have been renamed to include the name of the corresponding agent. The name of the endpoint registry is the new name from your Powerpoint, AAS registry.

**Pull request merged**